commitment/commitment_test.go

```diff
@@ -1476,7 +1476,7 @@ func TestTapCommitmentDeepCopy(t *testing.T) {
 	)
 }
 
-// TestTaprootAssetCommitmentScript tests that we're able to properly verify if
+// TestIsTaprootAssetCommitmentScript tests that we're able to properly verify if
```

Comment:
The linter isn't happy about this line that's now too long.

Reply:
@crystalstall 
Wrap text s.t. 80 characters is the max line length and the linter CI `lll` will pass, if you would please.
Then merge can happen